runtime/compiler/z/codegen/J9UnresolvedDataReadOnlySnippet.cpp

```diff
@@ -89,26 +91,25 @@ J9::Z::UnresolvedDataReadOnlySnippet::emitSnippetBody()
    TR::GlobalFunctionCallData dataDestination(glueSymRef, getNode(), cursor, cg(), self());
    cursor = cg()->getObjFmt()->encodeGlobalFunctionCall(dataDestination);
 
-   // Relative address to the CCUnresolvedData*
```

Comment:
We should probably leave this comment in.

Reply:
Added the comment back in https://github.com/eclipse/openj9/compare/ee40f0a3614cab3847b46a4b61c6fb2795d4b775..3ac390dbe99682be85f71980c4568d19c47c35dd